source/manual/cookie-consent-on-govuk.html.md

```diff
@@ -121,7 +121,7 @@ Regardless of how your cookie is set, you need to do the following:
 
 [cookie settings page]: https://www.gov.uk/help/cookies
 [Cookie settings page in Frontend GitHub repo]: https://github.com/alphagov/frontend/blob/master/app/views/help/cookie_settings.html.erb
-[Page template which pulls in the cookie banner]: https://github.com/alphagov/static/blob/54706a6eddcf71e2d6cd36b3239798293530d4e6/app/views/layouts/govuk_template.html.erb#L50
-[Cookie banner component]: https://govuk-publishing-components.herokuapp.com/component-guide/cookie_banner
+[Page template which pulls in the cookie banner]: https://github.com/alphagov/govuk_publishing_components/blob/master/app/views/govuk_publishing_components/components/_layout_for_public.html.erb#L86
+[Cookie banner component]: https://github.com/alphagov/govuk_publishing_components/blob/master/app/views/govuk_publishing_components/components/_cookie_banner.html.erb
```

Comment:
Yeah, have thought about that (as it would be a direct correspondent to the old Heroku link) but then, for consistency, we shall be doing the same for the page template link above (point it to https://components.publishing.service.gov.uk/component-guide/layout_for_public). Not sure which one is more appropriate/useful to be honest 🤷  happy to follow your suggestion

Reply:
Maybe they're both useful? Could we include both?

Reply:
Have added [an extra commit](https://github.com/alphagov/govuk-developer-docs/pull/3419/commits/141ad975d2afc0f900ab7aca42bf61ff5ca8c106?short_path=114579f#diff-114579fde79505f018c44bc2069cff02fd6069a696bbe3cd51b4d58cb5268f0b) to consistently point to both the guide and the code

Reply:
That's really good - I like it.